components/header.tsx

```diff
@@ -18,7 +26,24 @@ export default function Header() {
           </a>
         </Link>
       </Flex>
-      <Flex as="nav" sx={{ justifyContent: "center" }}>
+      <Flex
+        as="nav"
+        className={isActive ? "d-flex" : ''}
+        sx={{
+          justifyContent: "center",
+          [MEDIA_QUERY.mobile]: {
+            display: 'none',
+            position: 'fixed',
```

Comment:
maybe to solve the scrolling issue we can just make it position sticky and it might be ok, not sure I would need to try. But position sticky and top: 0 will make this go up to the top when scrolling. Not sure if the menu is bigger than the screen though you might need a double scroll and it will be ugly